development/docker-compose-deps.yml

```diff
@@ -19,3 +19,35 @@ services:
       interval: 5s
       timeout: 5s
       retries: 3
+  task-manager:
+    image: "${TASK_MANAGER_DOCKER_IMAGE:-prefecthq/prefect:3.0-python3.12}"
+    command: prefect server start --host 0.0.0.0 --ui
+    environment:
+      PREFECT_API_DATABASE_CONNECTION_URL: postgresql+asyncpg://postgres:postgres@task-manager-db:5432/prefect
+      PREFECT_LOCAL_STORAGE_PATH: ${PREFECT_LOCAL_STORAGE_PATH:-/opt/prefect/}
```

Comment:
Shouldn't we also use the same docker volume as the other containers? Otherwise I think we cannot see run states inside the Prefect UI.